example-apps/todo-mvc-example/source/app.ts

```diff
@@ -1,14 +1,22 @@
-import {bootstrap, bind} from 'angular2/angular2';
+// Load Global Styles
+import 'todomvc-common/base.css';
+import 'todomvc-app-css/index.css';
+
+import {bootstrap} from 'angular2/platform/browser';
```

Comment:
I think it's probably better to import this like 'import { bootstrap } from 'angular2/bootstrap';
I believe it all comes from the same place, but this is in the official examples. Am i missing something else about this import?